includes/login_state.js

```diff
@@ -1,27 +0,0 @@
-LoginState = {};
```

Comment:
new file is `client/login_state.js. also changed on bower and node package.json files